src/main/java/uk/gov/companieshouse/search/api/mapper/ApiToResponseMapper.java

```diff
@@ -33,16 +33,20 @@
                 return ResponseEntity.status(BAD_REQUEST).build();
             case DATE_FORMAT_ERROR:
                 return ResponseEntity.status(BAD_REQUEST)
-                        .body("Date provided is either invalid, empty or in the incorrect format, please use the format of 'yyyy-mm-dd' e.g '2000-12-20'");
+                        .body("Date provided is either invalid, empty or in the incorrect format, " +
+                            "please use the format of 'yyyy-mm-dd' e.g '2000-12-20'");
             case MAPPING_ERROR:
                 return ResponseEntity.status(BAD_REQUEST)
-                    .body("Error attempting to map request parameter values, please check the values of fields such as 'company_status' contain accurate values");
+                    .body("Error attempting to map request parameter values, please check the values of fields " +
+                        "'company_status' or `company_type` contain accurate values");
```

Comment:
should use the same style of apostrophe for both those surrounding company_status and company_type